web/src/app/schedules/on-call-notifications/ScheduleOnCallNotificationsEditDialog.tsx

```diff
@@ -32,7 +26,7 @@ export default function ScheduleOnCallNotificationsEditDialog(
       ? DateTime.fromFormat(rule.time, 'HH:mm', { zone }).toISO()
       : null,
     weekdayFilter: rule?.time ? rule.weekdayFilter || EVERY_DAY : NO_DAY,
-    type: channelTypeFromTarget(rule?.target),
+    type: rule?.target?.type ?? 'slackChannel',
```

Comment:
Does the type of slack channel no longer come in? If there's no type it seems odd to assume slackChannel

I feel like this may produce an issue in the future if we add more types- the code before was explicit in what types come in using a switch statement